crates/scroll/chainspec/src/constants.rs

```diff
@@ -1,9 +1,14 @@
 use crate::genesis::L1Config;
+use alloy_eips::eip1559::BaseFeeParams;
 use alloy_primitives::{address, b256, Address, B256};
 
 /// The transaction fee recipient on the L2.
 pub const SCROLL_FEE_VAULT_ADDRESS: Address = address!("5300000000000000000000000000000000000005");
 
+/// The system contract on L2 mainnet.
+pub const SCROLL_MAINNET_SYSTEM_CONTRACT_ADDRESS: Address =
```

Comment:
```suggestion
pub const SCROLL_MAINNET_L2_SYSTEM_CONFIG_CONTRACT_ADDRESS: Address =
```

A bit verbose, but "system contract" made me think of Ethereum's system contracts.